src/excalidraw-utils/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@excalidraw/utils",
+  "version": "0.7.0",
```

Comment:
`0.1.0` ?

Reply:
Good question!
Do we have a base version for Excalidraw? Do we want to keep the version in-sync? If not, then you are probably right we should use `0.1.0`.

Reply:
We haven' yet released the version of `excalidraw` from this repo. The version bump is incremented based on [excalidraw-embed](https://github.com/excalidraw/excalidraw-embed). 
And since this both are independent packages and will be published independently we can start with `0.1.0` else it will be confusing to users also coz prev versions doesn't have any existence.